README.md

```diff
@@ -85,8 +85,9 @@ We will backport bugfixes--but not new features--into older versions of
 | client-go 1.4       | +              | ✓              | -              | -              | -              |
 | client-go 1.5       | +              | +              | -              | -              | -              |
 | client-go 2.0       | +              | +              | ✓              | -              | -              |
-| client-go 3.0       | †              | +              | +              | ✓              | -              |
-| client-go HEAD      | †              | †              | +              | +              | +              |
+| client-go 3.0       | †              | †              | †              | ✓              | -              |
```

Comment:
Explained IRL, it's not a bug. 

extensions/v1beta1.Job was removed in kubernetes release-1.6, thus the dagger signs for client-go after 3.0.

Kubernetes 1.5 did announce a [deprecation notice](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md#deprecations-2). client-go 2.0 should had copied that notice to give people 1 release to adapt.